docs/medical-api/api-reference/patient/list-patients.mdx

```diff
@@ -6,13 +6,14 @@ api: "GET /medical/v1/patient"
 
 ## Query Params
 
-<ParamField query="facilityId" type="string" required>
+<ParamField query="facilityId" type="string" Optional>
```

Comment:
Depending on suggestion

Reply:
I don't understand what you mean. Maybe forgot to add another comment to the PR which this would refer to? 😃

Reply:
Oh sorry it's in reference to this comment https://github.com/metriport/metriport/pull/369#discussion_r1199510950